### ISSUE TITLE
Missing Vulkan 1.1 functions

### ISSUE THREAD

**Issue by kjozic** [vulkan-go/vulkan#33]
Vulkan 1.1 introduced some new functions. The most important function is EnumerateInstanceVersion() which is used to enumerate supported instance versions. Many extensions are now members of the Vulkan core:
- VK_KHR_16bit_storage
- VK_KHR_bind_memory2
- VK_KHR_dedicated_allocation
- VK_KHR_descriptor_update_template
- VK_KHR_device_group
- VK_KHR_device_group_creation
- VK_KHR_external_memory
- VK_KHR_external_memory_capabilities
- VK_KHR_external_semaphore
- VK_KHR_external_semaphore_capabilities
- VK_KHR_external_fence
- VK_KHR_external_fence_capabilities
- VK_KHR_get_memory_requirements2
- VK_KHR_get_physical_device_properties2
- VK_KHR_maintenance1
- VK_KHR_maintenance2
- VK_KHR_maintenance3
- VK_KHR_multiview
- VK_KHR_relaxed_block_layout
- VK_KHR_sampler_ycbcr_conversion 
- VK_KHR_shader_draw_parameters
- VK_KHR_storage_buffer_storage_class
- VK_KHR_variable_pointers

Vulkan 1.1 compatibility statement can be found [here](https://www.lunarg.com/wp-content/uploads/2018/05/1.1-Compatibility-Statement_05_18.pdf). Most important changes are described [here](https://vulkan.lunarg.com/doc/sdk/1.1.85.0/windows/tutorial/html/index.html).

Do you plan to implement missing functions?

**Comment by xlab** [comment]
@kjozic yep, it seems that I updated to 1.1 without capturing those functions, I will refactor the API to match 1.1 better soon.

**Comment by yarbelk** [comment]
@xlab any news on this?

**Comment by cers000** [comment]
The Vulkan 1.2 specification was launched on January 15th, 2020

**Comment by einthusan** [comment]
A repo called "go-vulkan" seems to support Vulkan 1.1 and has a much neater API than this repo (vulkan-go).
https://github.com/sparkkoori/go-vulkan

Is anyone willing to do a performance test comparing the two packages?

**Comment by xlab** [comment]
Keeping open for visibility and bookkeeping. I hope this doesn't cause distraction.
